go.mod

```diff
@@ -20,29 +16,26 @@ require (
 	github.com/go-openapi/strfmt v0.19.0 // indirect
 	github.com/go-openapi/swag v0.19.0 // indirect
 	github.com/go-openapi/validate v0.19.0 // indirect
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
-	github.com/hashicorp/go-getter v0.0.0-20180410174945-b345bfcec894 // indirect
-	github.com/hashicorp/go-hclog v0.0.0-20180402200405-69ff559dc25f // indirect
-	github.com/hashicorp/go-multierror v0.0.0-20171204182908-b7773ae21874 // indirect
-	github.com/hashicorp/go-plugin v0.0.0-20180331002553-e8d22c780116 // indirect
+	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
+	github.com/hashicorp/atlas-go v0.0.0-20161107204910-1792bd8de119 // indirect
+	github.com/hashicorp/go-immutable-radix v1.0.0 // indirect
+	github.com/hashicorp/go-sockaddr v1.0.0 // indirect
 	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce // indirect
-	github.com/hashicorp/hcl2 v0.0.0-20180406023453-9db880accff1 // indirect
-	github.com/hashicorp/terraform v0.11.12-beta1.0.20190214175014-182daa61982
+	github.com/hashicorp/serf v0.8.2-0.20171022020050-c20a0b1b1ea9 // indirect
+	github.com/hashicorp/terraform v0.12.0
```

Comment:
it would be best to vendor v0.12.2, the latest release as of commenting.
```
GO111MODULE=on go get github.com/hashicorp/terraform@v0.12.2
GO111MODULE=on go mod tidy
GO111MODULE=on go mod vendor
```

Reply:
several opaque dependency issues trying to vendor 0.12.2. Went with 0.12.1 which worked without issue.